en/docs/references/call-a-ballerina-library-function.md

```diff
@@ -0,0 +1,49 @@
+# Call a Ballerina Library function using the Statement Editor
+
+Statement-editor allows you to easily navigate the Ballerina libraries which include Ballerina standard libraries and Ballerina language libraries. The **libraries** tab in Statement-editor lists all the supported standard and language libraries.
+
+This guide helps you to understand	how to call a Ballerina library functions using the statement-editor.
+
+Let's call the Ballerina [printError](https://lib.ballerina.io/ballerina/log/2.5.0/functions#printError) function of the [log module](https://lib.ballerina.io/ballerina/log/2.5.0) to log an error in the given sample code below
+
+```
+public function main() returns error? {
+
+    error e = error("something went wrong!");
+    // Add the function call to log and error
+}
+
+```
+
+To build the log statement, follow these steps:
+
+1. On the low-code diagram, click **+** within the main method. Then click **Function** in the **Add Constructs** pane that opens.
+
+    ![Add function statement](../img/statement-editor/add-function-statement.gif)
+
+    The libraries tab will be selected by default listing all the supported Ballerina libraries
+
+2. Search for **log** and select log module. This will list down all the supported functions, errors, records, type, etc.. in the log module. Select **log:printError** function.
```

Comment:
```suggestion
2. Search for **log** and select the `log` module. This will list all the supported functions, errors, records, types, etc. of the `log` module. Select the **log:printError** function.
```